front_end/src/components/Tiles/TileForm/TileForm.js

```diff
@@ -188,6 +188,14 @@ export default class TileForm extends Component {
                     <option value="sponsored">Sponsored</option>
                   </select>
                 </div>
+                <div className="form-group">
+                  <label htmlFor="TilePosition">Position Priority</label>
+                  <select className="form-control" onChange={this.handleChange} id="TilePosition" name="position_priority" ref="position_priority" defaultValue={data.position_priority} >
+                    <option value="medium">medium</option>
+                    <option value="low">low</option>
+                    <option value="high">high</option>
```

Comment:
should these be in a more logical order? low to high or high to low? the default can still be medium.